src/PrestaShopBundle/Resources/config/services/core/configuration.yml

```diff
@@ -2,8 +2,9 @@ services:
   _defaults:
     public: true
 
+  PrestaShop\PrestaShop\Core\Configuration\IniConfiguration: ~
   prestashop.core.configuration.ini_configuration:
-    class: 'PrestaShop\PrestaShop\Core\Configuration\IniConfiguration'
+    alias: 'PrestaShop\PrestaShop\Core\Configuration\IniConfiguration'
```

Comment:
Good call! but can you add the deprecation of this alias while we're at it please?

Reply:
```suggestion
    alias: 'PrestaShop\PrestaShop\Core\Configuration\IniConfiguration'
    deprecated:
      package: PrestaShop\PrestaShop
      version: 9.0

```